.github/workflows/publish.yml

```diff
@@ -26,3 +25,14 @@ jobs:
         uses: katyo/publish-crates@v1
         with:
           registry-token: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+  create-release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Create GitHub release
+        uses: ncipollo/release-action@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I see the `create-release` action offers the option to also upload an artifact,  and it makes me wonder if that would useful to provide a tar binary per release?

Reply:
I do like this idea. I'm not entirely sure how to implement ort though. We'd want to provide artifacts for multiple platforms, I presume. They would be different jobs (in a matrix?), that all contribute an artifact to the release. I don't know if that would work with this plugin. Perhaps I'll do an experiment tomorrow to see if the runs can be made to combine, or they will conflict.

Reply:
I believe the we're doing release artefacts in tree-sitter so that the `npm` installer has something easy to download and install.  But this crate is largely installed via `cargo install`, and so I think it's fine to create the release without uploading anything to it.  Put another way, publishing to crates.io is the equivalent to / replacement for attaching a tarball to the GitHub release.

Reply:
In that case, let's not put the effort for now.